src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestMessage.cs

```diff
@@ -115,6 +115,9 @@ public HttpMethod Method
         [Obsolete("HttpRequestMessage.Properties has been deprecated. Use Options instead.")]
         public IDictionary<string, object?> Properties => Options;
 
+        /// <summary>
+        /// Gets the options for the HTTP request.
```

Comment:
I'd expect this to get a bit more specific description so that users actually know what it's good for a and how to use it. 
But the whole `Options` related docs are WASM addition so it's up them how much details they want to put in, cc @lewing.